api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceTest.java

```diff
@@ -66,12 +71,46 @@
     @Autowired
     private AppFrameworkConfig appFrameworkConfig;
 
+    @Autowired
+    private static AllAppDescriptors allAppDescriptors;
+
+    @Autowired
+    private static AllFreeStandingExtensions allFreeStandingExtensions;
+
 	@Before
 	public void setup() throws IOException {
         //trigger loading of the apps
 		new AppFrameworkActivator().contextRefreshed();
 	}
 
+	private User saveUser(UserService us, User u, String password)
+	{
+		User user = null;
+		Class<?> userServiceClass = us.getClass();
+    	try {
+    		// openmrs 1.9
+    		Method saveUserMethod =  userServiceClass.getDeclaredMethod("saveUser", u.getClass(), password.getClass());
+    		user = (User)saveUserMethod.invoke(us, u, password);
+    		return user;
+    	} catch (NoSuchMethodException ex) {
```

Comment:
I know we normally do this but i think it's cleaner to just catch ReflectiveOperationException since it's a common super class